go/developer/web.config

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <system.webServer>
+        <rewrite>
+            <rules>
+                <clear />
+
+                <!-- Links for Developer 10.0 onward -->
+
+                <rule name="/go/developer/X.0/help-keyboards" stopProcessing="true">
+                  <match url="^(10|11|12|13)\.0/help-keyboards" />
```

Comment:
Let's make the match on `([1-9][0-9])\.[0-9]` so we don't need to update this every release.